javalib/src/main/scala/java/time/temporal/ChronoField.scala

```diff
@@ -0,0 +1,151 @@
+package java.time.temporal
+
+final class ChronoField private (name: String,
+                                 ordinal: Int,
+                                 val range: ValueRange,
```

Comment:
Same remark as before: there should be a separate `def range()`. There might be other occurrences.